docs/Networking.asciidoc

```diff
@@ -154,8 +156,19 @@ IPADDR='10.0.2.2/15'
 STARTMODE='auto'
 ---------------
 
-wicked 0.6.23 and later has enhanced support for the creation and configuration of OpenvSwitch bridges,
-so the permanent configuration should look like this:
+wicked 0.6.23 and later has enhanced support for the creation and configuration of OpenvSwitch bridges. 
+But ensure that the OpenvSwitch service starts before the Network service by verifying/editing the OpenvSwitch service file: 
+
+[source, bash]
+---------------
+ #/usr/lib/systemd/system/openvswitch.service
+ [Unit]
+ Description=Open vSwitch
+ Before=network.service
+ ...
```

Comment:
@coolo are you using ifcfg-brX for automatic openvswitch start and ifcfg-tapX for the tap devices on workers? This change of osv service file is needed in my case (Leap installed last week inc. updates), otherwise bridge device has no IP address and os-autoinst-openswitch won't start after reboot.